apps/condo/domains/common/utils/mutations.utils.ts

```diff
@@ -137,6 +142,15 @@ function runMutation ({ action, mutation, variables, onCompleted, onError, onFin
                                     name: fieldPath[fieldPath.length - 1],
                                     errors: [messageForUser],
                                 })
+                            } else if (OnErrorMsg === null) {
+                                const type = get(error, ['extensions', 'type'])
+                                const message = get(commonErrorTypesToMessagesMapping, type)
+                                if (message) {
+                                    notification.error({
+                                        message: commonErrorTypesToMessagesMapping[type],
```

Comment:
better to format it to minutes: istead of wait for 120 seconds -> wait for 2 minutes

Reply:
+1 for that

Reply:
+1

Reply:
Or you can just say something like "Try again later"

Reply:
I've hide text with seconds
<img width="735" alt="image" src="https://github.com/user-attachments/assets/cca3e177-e265-4fa9-a92a-3a7fea3f32b7" />